modules/pubxaiAnalyticsAdapter.js

```diff
@@ -134,8 +140,9 @@ pubxaiAnalyticsAdapter.shouldFireEventRequest = function (samplingRate = 1) {
 function send(data, status) {
   if (pubxaiAnalyticsAdapter.shouldFireEventRequest(initOptions.samplingRate)) {
     let location = utils.getWindowLocation();
-    if (typeof data !== 'undefined' && typeof data.auctionInit !== 'undefined') {
```

Comment:
any reason why you took the `data.auctionInit !== undefined` check away?

Reply:
I have removed auctionInit from the data object completely as we don't need it anymore. That's why this condition isn't required here.